src/fastmcp/utilities/types.py

```diff
@@ -31,6 +31,10 @@
 NotSetT: TypeAlias = EllipsisType
 
 
+def get_fn_name(fn: Callable[..., Any]) -> str:
```

Comment:
The things we do for type completeness

Reply:
Next PR will need to include `get_name_of_fn_name_fn`